### PR TITLE
Readme bin2tiff

### PR DESCRIPTION
**Pull request by weiqin61**
Add the information from https://github.com/terraref/documentation/pull/164/files to https://github.com/terraref/documentation/blob/4148708f2c672ff90a3ce356547a22608b50b208/bin2tiff_readme.md
Now the new README has 3 categories.

### CODE REVIEW

File: bin2tif/readme.md [hunk 1]
```diff
@@ -14,6 +15,80 @@ _Output_
...
+   ```
+        i. Convert coordinates from Scanalyzer to MAC coordinates using formula from [https://terraref.gitbooks.io/terraref-documentation/content/user/geospatial-information.html](https://terraref.gitbooks.io/terraref-documentation/content/user/geospatial-information.html)
+
+        ii. Use utm tools to convert coordinates from MAC to lat/lon
+   ```
+   e. Using osgeo.gdal, associate with geo-bounding box, create geotiff.
+
+### Parameters
+
+HEIGHT_MAGIC_NUMBER and PREDICT_MAGIC_SLOPE were applied when we estimate the field of view, this is a testing based number.
```
Comment (dlebauer) [inline bin2tif/readme.md]: @ZongyangLi when you say in the README "HEIGHT_MAGIC_NUMBER and PREDICT_MAGIC_SLOPE were applied when we estimate the field of view, this is a testing based number." ... is it correct to say that these numbers are empirically derived based on some calibration? Do you have additional information about how these numbers are derived?

File: bin2tif/readme.md [hunk 2]
```diff
@@ -14,6 +15,80 @@ _Output_
...
+
+The geo-reference bounding box is based on an assumption that image is aligned to geographic coordinates, so that moving up in the image corresponds to moving exactly north.
```
Comment (dlebauer) [inline bin2tif/readme.md]: @ZongyangLi do you have any estimate of how much error this introduces over the image area?

File: bin2tif/readme.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # Bin2Tif Extractor
```
Comment (dlebauer) [inline bin2tif/readme.md]: missing authors ... I will add these

File: bin2tif/readme.md (hunk 1 above)
Comment (ZongyangLi) [reply bin2tif/readme.md]: Background: when I use FOV in metadata, I am not able to get a well stitched full field map.

Steps for creating these MAGIC NUMBER:
1. Make an assumption that the real FOV can be derived as a linear function base on the given FOV.
2. Randomly pick several different days stereoTop RGB raw data in different camera height, try to get a best stitched full field map for each day empirically(by personal judgement), and record all the FOV for each day.
3. Apply a linear regression to output HEIGHT_MAGIC_NUMBER and PREDICT_MAGIC_SLOPE

File: bin2tif/readme.md (hunk 2 above)
Comment (ZongyangLi) [reply bin2tif/readme.md]: I am sorry I do not have it yet. Further investigation might be done if you think it is valuable.

File: bin2tif/readme.md [hunk 4]
```diff
@@ -66,3 +141,4 @@ qsub /projects/arpae/terraref/shared/extractors/extractors-stereo-rgb/bin2tif/ba
 * All the Python scripts syntactically support Python 2.7 and above. Please make sure that the Python in the running environment is in appropriate version.
```
Comment (dlebauer) [inline bin2tif/readme.md]: @ZongyangLi does this refer to all of the code in here? Does it imply it works with Python 3.x?

File: bin2tif/readme.md [hunk 5]
```diff
@@ -14,6 +15,80 @@ _Output_
...
+    2. for the ground, we judged that small stitching errors were not worth the (substantial) additional effort to build the more complete model.
+
+### Failure Conditions
+
+* If the camera is moved in the gantry box, then the magic numbers may have to be recalculated or experimentally determined.
+
+* If the camera is not aligned north-south, then the geo-bounding box may not be accurate.
+
+
+## Implementation
```
Comment (dlebauer) [inline bin2tif/readme.md]: @weiqin61 and/or @ZongyangLi it would be nice to have a list of files and what they do

File: bin2tif/readme.md (hunk 4 above)
Comment (ZongyangLi) [reply bin2tif/readme.md]: It should be work for Python 3.x, but Python 2.7 is recommended.

File: bin2tif/readme.md (hunk 2 above)
Comment (dlebauer) [reply bin2tif/readme.md]: That is okay. I think that the error is on the order of inches over the field, which means that the error in the position of a pixel in opposite corners of the image is << the resolution of the image.

**Pull request merged**